docs/source/user-guide/nlp/supported_tasks.rst

```diff
@@ -0,0 +1,100 @@
+  to a document) is assigned a class (in the binary/multiclass case) or a certain set of classes (in the multilateral
+  case). In both the binary, the multiclass and the multilabel case the class "belongs" / "classifies" the whole text
+  sample. Examples for such tasks are Sentiment Analysis, Topic Extraction, detecting harmful content and more.
+* **Token Classification**: Token Classification is any NLP task in which each word (or to be more accurate, token) in
+  the text sample is assigned a class of it's own. In many cases most tokens will belong to a "background" class,
+  allowing the model to focus on the interesting tokens. Examples for such tasks are Named Entity Recognition, Part
+  of Speach annotation (in which all tokens have a non-background class).
+
+.. _supported_labels__predictions_format:
+
+Supported Labels and Predictions Format
+=======================================
+
+While labels are passed while constructing the :class:`TextData <text_data.TextData>` object, predictions are passed
+separately to the ``run`` method of the check / suite. Labels and predictions must be in the format detailed in this
+section, according the task type.
+
+Label Formats:
+--------------
+
+* **Text Classification** Label - For text classification the accepted label format differs between multilabel and
+  single label cases. For single label data, the label should be passed as a sequence of labels, with one entry
+  per sample that can be either a string or an integer. For multilabel data, the label should be passed as a
+  sequence of sequences, with the sequence for each sample being a binary vector, representing the presence of
+  the i-th label in that sample.
+* **Token Classification** Label - For token classification the accepted label format is a sequence of sequences,
+  with the inner sequence containing tuples in the following format: (class_name, span_start, span_end).
+  span_start and span_end are the start and end character indices of the token within the text, as it was
+  passed to the raw_text argument. Each upper level sequence contains a sequence of tokens for each sample.
+
+>>> text_classification_label_multiclass = [0, 0, 1, 2]
+>>> text_classification_label_multilabel = [[0, 0, 1], [0, 1, 1], [1, 0, 1], [0, 0, 0]]
+>>> token_classification_label = [[('class_1', 0, 2), ('class_2', 7, 10)], [('class_2', 42, 54)], []]
+
+Prediction Formats:
+-------------------
+
+The accepted formats for providing model predictions and probabilities are detailed below
+
+**Text Classification**
+
+*Single Class Predictions*
+
+* predictions - A sequence of class names or indices with one entry per sample, matching the set of classes
+  present in the labels.
+* probabilities - A sequence of sequences with each element containing the vector of class probabilities for
+  each sample. Each such vector should have one probability per class according to the class (sorted) order, and
```

Comment:
not sure - what is the "sorted" order? (who sorts this? confusing a bit, is there anything I should actively do to verify consistency with their "order" somewhere else? or just have a consistent order with myself? and are the class names displayed somewhere? etc.)